===================================================================
RCS file: /cvs/gnome/ooo-build/patches/vclplug/Attic/vcl-gtk-mainloop-m79.diff,v
retrieving revision 1.1.2.3
retrieving revision 1.1.2.4
diff -u -r1.1.2.3 -r1.1.2.4
--- patches/vclplug/vcl-gtk-mainloop-m79.diff	2005/04/04 14:14:54	1.1.2.3
+++ patches/vclplug/vcl-gtk-mainloop-m79.diff	2005/06/23 11:52:33	1.1.2.4
@@ -120,7 +120,21 @@
 diff -u -p -u -r1.4 conditn.c
 --- sal/osl/unx/conditn.c	16 Mar 2001 13:14:43 -0000	1.4
 +++ sal/osl/unx/conditn.c	10 Mar 2005 10:11:43 -0000
-@@ -278,7 +278,7 @@ oslConditionResult SAL_CALL osl_waitCond
+@@ -234,13 +234,6 @@ sal_Bool SAL_CALL osl_resetCondition(osl
+ 	}
+ 
+ 	pCond->m_State = sal_False;
+-    nRet = pthread_cond_broadcast(&pCond->m_Condition);
+-	if ( nRet != 0 )
+-	{
+-       OSL_TRACE("osl_resetCondition : condition broadcast failed. Errno: %d; %s\n",
+-                  nRet, strerror(nRet));
+-		return sal_False;
+-	}
+ 
+     nRet = pthread_mutex_unlock(&pCond->m_Lock);
+     if ( nRet != 0 )
+@@ -278,7 +271,7 @@ oslConditionResult SAL_CALL osl_waitCond
  		return osl_cond_result_error;
  	}
  
